arrow_pd_parser/parse.py

```diff
@@ -66,18 +88,23 @@ def pa_read_json(json_path, parse_options=None, read_options=None):
     if read_options is None:
         read_options = {}
 
-    json_parse = json.ParseOptions(**parse_options)
+    if ("explicit_schema" in parse_options) and (schema is not None):
+        raise KeyError(
+            "column_types cannot be set as schema parameter"
+            "is passed to this parameter in the Convert Option"
```

Comment:
need a space between "parameter" and "is". Also this message could use some rephrasing (or at least some punctuation), as I had to read it a couple of times to get what it means. What I think you mean to say is something like, "you should/must not include `column_types` in `parse_options`, as these will already be dealt with in the supplied schema"

Reply:
@Thomas-Hirsch have pushed updated